mllib/src/main/scala/org/apache/spark/ml/evaluation/MulticlassClassificationEvaluator.scala

```diff
@@ -38,6 +38,7 @@ class MulticlassClassificationEvaluator (override val uid: String)
   /**
    * param for metric name in evaluation (supports `"f1"` (default), `"precision"`, `"recall"`,
    * `"weightedPrecision"`, `"weightedRecall"`)
+   * Default: f1
```

Comment:
This already says the default 2 lines above.  Keep only one statement of the default.